src/imitation/scripts/common/rl.py

```diff
@@ -117,6 +123,13 @@ def make_rl_algo(
         if rl_kwargs.get("batch_size") is not None:
             raise ValueError("set 'batch_size' at top-level")
         rl_kwargs["batch_size"] = batch_size
+        if relabel_reward_fn:
+            assert not isinstance(venv.observation_space, gym.spaces.Dict)
+            # Note(yawen): We don't support HerReplayBuffer or DictReplayBuffer yet,
+            # since current RewardFn only takes in NumPy array-based inputs.
+            rl_kwargs["replay_buffer_class"] = ReplayBufferRewardWrapper
```

Comment:
What if `rl_kwargs["replay_buffer_class"]` is already specified? Should we save this and pass it through as `replay_buffer_cls` in `replay_buffer_kwargs`?

Reply:
Yep, I think you're correct, although this could be silent.